components/TOC.js

```diff
@@ -10,7 +10,7 @@ export default function TOC({
 }) {
   if (!toc || !toc.length) return null
   const minLevel = toc.reduce((mLevel, item) => (!mLevel || item.lvl < mLevel) ? item.lvl : mLevel, 0)
-  const tocItems = toc.filter(item => item.lvl <= minLevel + 2).map(item => ({ ...item, content: item.content.replace(/[\s]?\{\#[\w\d\-_]+\}$/, '') }))
+  const tocItems = toc.filter(item => item.lvl <= minLevel + 2).map(item => ({ ...item, content: item.content.replace(/[\s]?\{\#[\w\d\-_]+\}$/, '').replace(/<a name="[\w\d\-_]*"><\/a>/, '') }))
```

Comment:
You can use the regex to remove all HTML tags, not only `<a>` :)

```suggestion
  const tocItems = toc.filter(item => item.lvl <= minLevel + 2).map(item => ({ ...item, content: item.content.replace(/[\s]?\{\#[\w\d\-_]+\}$/, '').replace(/(<([^>]+)>)/gi, '') }))
```

Reply:
oh, that's better. Thanks :)

Reply:
Please also test it on the `2.1.0` spec :)

Reply:
Working alright
![Screenshot](https://user-images.githubusercontent.com/54525741/122969755-a28d4380-d3aa-11eb-9e20-5da906fbbf09.png)

Reply:
@magicmatatjahu Here's the Screenshot in case you missed it. :)